### DISCUSSION TITLE
Question regardomg LCM lora for sd 1.5

### DISCUSSION THREAD

**Discussion by gsgoldma** [vladmandic/sdnext#2497] [Q&A]
It doesn't seem to be working for me. I downloaded it from Huggingface, the one titled pytorch model.safetensors. 

Is it for diffusers only models? If so, is there one that works with original.backend too?

**Comment by vladmandic** [comment]
No. It's covered in readme.

**Reply by gsgoldma** [reply]
Should the LCM sampler always be available? It's not showing up in settings under possible samplers.

**Reply by vladmandic** [reply]
It's diffusers only.

**Reply by gsgoldma** [reply]
so is the lora only compatible with diffuser models downloaded from huggingface, and not the models from civitai?

And, how do we load a diffuser lora? It doesn't seem to be in extra neworks, and when it's downloaded, it seems to be in the base models folder

Edit: sometimes I get this error too when trying the LCM dreamshaper model, even though sometimes it works: cuFFT doesn't support signals of half type with compute capability less than SM_53, but the device containing input half tensor only has SM_52

**Reply by vladmandic** [reply]
lcm lora is just a lora and can be applied on top of any model. download and place to models/lora as usual.
but lcm scheduler is only available via diffusers. so using lcm lora in original backend is - well, a mixed bag.

**Reply by gsgoldma** [reply]
I was confused because I thought the LCM Lora also required the LCM Scheduler, but I'm guessing that's not true now that you've said that. Thanks for clarifying.

**Reply by vladmandic** [reply]
lcm lora is just lora. but what will it do without lcm scheduler is anyone's guess - its designed to work with lcm scheduler. it doesn't mean that it cannot work with others - that's the beauty of ml world, we can experiment with a lot of different combos.
